repose-aggregator/components/services/atom-feed-service/atom-feed-service-api/build.gradle

```diff
@@ -0,0 +1,12 @@
+jacocoTestCoverageVerification {
+    violationRules {
+        rules.forEach { rule ->
+            rule.limits.forEach { limit ->
+                if (limit.counter == 'INSTRUCTION' && limit.value == 'COVEREDRATIO') {
+                    // todo: set to the current coverage -- need more coverage, but we don't want the build to fail for the baseline
+                    limit.minimum = 0.0
```

Comment:
???

Reply:
Ah. The API has some testable classes, but no tests in the project. All of the tests are in the impl project. So when run on its own, this project reports a coverage of 0%. That's not actually the case, though, since those classes are used in tests elsewhere.

I must have gotten a bit trigger-happy when going down the list to have missed that. However, the reason I hit this is probably because it failed a build, which means that the test coverage from the other project is not being counted here. I... am not entirely sure how to fix that. I could ignore api projects, but then we don't have numbers for things in API projects (e.g., Exceptions, enums). Or we could add tests to API projects, but... they would probably be really simple, and kind of pointless, tests. Ideally, we would be able to aggregate JaCoCo coverage metrics and act on the aggregation rather than each individual project, but support for doing so does not really exist in the JaCoCo plugin. We could probably come up with a way to do that (maybe like this: https://gist.github.com/aalmiray/e6f54aa4b3803be0bcac) but it's still not super pretty. Perhaps that is an enhancement we could save for a follow-up story?